semanticdb/scalac/library/src/main/scala/scala/meta/internal/semanticdb/scalac/DocumentOps.scala

```diff
@@ -12,16 +12,17 @@ trait DocumentOps { self: DatabaseOps =>
     if (!g.settings.Yrangepos.value) {
       sys.error("the compiler instance must have -Yrangepos enabled")
     }
-    if (g.useOffsetPositions) {
-      sys.error("the compiler instance must use range positions")
-    }
+    // for some reason this fails with ammonite, even though the first Yrangepos == true
+//    if (g.useOffsetPositions) {
+//      sys.error("the compiler instance must use range positions")
+//    }
     if (!g.settings.plugin.value.exists(_.contains("semanticdb"))) {
       sys.error("the compiler instance must use the semanticdb plugin")
     }
-    if (!g.analyzer.getClass.getName.contains("HijackAnalyzer")) {
-      println(g.analyzer.getClass.getName)
-      sys.error("the compiler instance must use a hijacked analyzer")
-    }
+    // in ammonite, this is ammonite.interp.CompilerCompatibility$$anon$2
```

Comment:
I'll try to have a look on the Ammonite side, this one could be that it highjacks the analyzer in turn